rules/test-title.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var util = require('../util');
+var literalTypes = ['Literal', 'TemplateLiteral'];
+
+module.exports = function (context) {
+	var ifMultiple = context.options[0] === 'if-multiple';
+	var isTestFile = false;
+	var testCount = 0;
+
+	return {
+		CallExpression: function (node) {
+			if (util.isTestFile(node)) {
+				isTestFile = true;
+				return;
+			}
+
+			if (isTestFile) {
+				var args = node.arguments;
+				var callee = node.callee;
+				var name = callee.type === 'MemberExpression' ? callee.object.name : callee.name;
+				if (name === 'test') {
+					testCount++;
+					var hasNoTitle = args.length !== 2 || literalTypes.indexOf(args[0].type) === -1;
```

Comment:
I think that some users (me first) would like a rule enforcing a "written" rule, that describes the tested behavior in a "string" format, rather than in a function name.

```js
// BAD
test(function shouldIncrementIfTypeIsINCREMENT(t) { ... });
test(function shouldDecrementIfTypeIsDECREMENT(t) { ... });

// GOOD
test('should increment value if type is INCREMENT', function(t) { ... });
test('should decrement value if type is DECREMENT', function(t) { ... });
```